src/model/request/loginRequest.ts

```diff
@@ -1,3 +1,22 @@
-export class LoginRequest {
-    constructor(public username?: string, public password?: string){ }
```

Comment:
Why did you remove the constructor?

Reply:
This thing make the Fre8back didnt compile before. Dont know why it work now. Fixed . Changed it back